packages/web/src/scss/helpers/index.html

```diff
@@ -3,11 +3,28 @@
   <section class="docs-Section">
 
     <h2 class="docs-Heading">Accessibility</h2>
-    <p>
-      The following element is visually hidden, but still accessible by AT (use dev tools to check it's really there!).
-    </p>
 
-    <p class="accessibility-hidden">This text is visually hidden</p>
+    <div class="mb-400">
+      <p>
+        The following element is visually hidden, but still accessible by AT (use dev tools to check it's really there!).
```

Comment:
I guess the period at the end of the line is redundant since there is exclamation mark inside the brackets. Perhaps you could move it to the next paragraph.

Reply:
> inside the brackets

Then it's not redundant. Or is there a rule that forbids that in English? 🤔